### PR TITLE
[FEAT] Is tradeable

### PR DESCRIPTION
**Pull request by adamhannigan**
# Description

Adds separate dates for when items should can be traded (in the marketplace) or withdrawn.

Includes some helpful labels so players know when an item is tradeable or withdrawable.

Screenshot 2024-12-13 at 3 09 59 PM
Screenshot 2024-12-13 at 3 09 08 PM

## How to test?

1. Connect to local API
2. Explore the marketplace and ensure only trade-able items are there
3. Go to withdraw collectibles + wearables and ensure the correct ones are showing

_Note: By default the items not ready for trading will not show in the marketplace list. You can manually change the URL using these item IDs to visit them though._

### CODE REVIEW

**Comment by brian-man** [comment]
Recommend: Use similar date formatting to the "What's On" section.  Meaning the "word" version of the month, potentially abbreviated to 3 letters.

At first look, I thought the two dates on the stocking were in different formats.

**Comment by adamhannigan** [comment]
@brian-man That makes sense 👍 

We mainly use `locale` strings, which in the above image uses the DD/MM/YYYY format. In the US I believe it would show as MM/DD/YYYY

File: src/features/marketplace/components/TradeableInfo.tsx [hunk 1]
```diff
@@ -113,11 +143,25 @@ export const TradeableDescription: React.FC<{
...
+          <div className="p-2 pl-0 pb-0 flex items-center justify-between">
             <Label type="danger" icon={SUNNYSIDE.icons.stopwatch}>
-              {t("marketplace.notForSale")}
+              {t("coming.soon")}
+            </Label>
+            <Label type="transparent">{formatDate(tradeAt)}</Label>
+          </div>
+        )}
+        {!canWithdraw && !!withdrawAt && (
+          <div className="p-2 pl-0 pb-0 flex items-center justify-between flex-wrap">
```
Comment (c-py) [inline src/features/marketplace/components/TradeableInfo.tsx:160]: desktop

These may need to flex-col on desktop view - getting a bit squashed

File: src/features/game/types/withdrawables.ts [hunk 2]
```diff
@@ -1,1781 +1,675 @@
...
+  "Striped Yellow Shirt": CAN_WITHDRAW_AND_TRADE,
+  "Paw Shield": CAN_WITHDRAW_AND_TRADE,
+  Pan: CAN_WITHDRAW_AND_TRADE,
+  "Royal Robe": CAN_WITHDRAW_AND_TRADE,
+  Crown: CAN_WITHDRAW_AND_TRADE,
+  "Soybean Onesie": CAN_WITHDRAW_AND_TRADE,
+  "Pharaoh Headdress": CAN_WITHDRAW_AND_TRADE,
+  "Scarab Wings": CAN_WITHDRAW_AND_TRADE,
+  "Fossil Head": CAN_WITHDRAW_AND_TRADE,
+
```
Comment (c-py) [inline src/features/game/types/withdrawables.ts:210]:
```suggestion
"Sunflower Amulet": CAN_WITHDRAW_AND_TRADE,
  "Beetroot Amulet": CAN_WITHDRAW_AND_TRADE,
  Parsnip: CAN_WITHDRAW_AND_TRADE,
  "Angel Wings": CAN_WITHDRAW_AND_TRADE,
  "Luna's Hat": CAN_WITHDRAW_AND_TRADE,
  Cattlegrim: CAN_WITHDRAW_AND_TRADE,
  "Ancient Rod": CAN_WITHDRAW_AND_TRADE,
  "Banana Onesie": CAN_WITHDRAW_AND_TRADE,
  "Beekeeper Hat": CAN_WITHDRAW_AND_TRADE,
  "Crimstone Amulet": CAN_WITHDRAW_AND_TRADE,
  "Hornet Mask": CAN_WITHDRAW_AND_TRADE,
  "Non La Hat": CAN_WITHDRAW_AND_TRADE,
  "Olive Shield": CAN_WITHDRAW_AND_TRADE,
  "Olive Royalty Shirt": CAN_WITHDRAW_AND_TRADE,
  "Oil Overalls": CAN_WITHDRAW_AND_TRADE,
  "Ancient Shovel": CAN_WITHDRAW_AND_TRADE,
  "Lemon Shield": CAN_WITHDRAW_AND_TRADE,
  "Green Amulet": CAN_WITHDRAW_AND_TRADE,
  "Devil Wings": CAN_WITHDRAW_AND_TRADE,
  "Eggplant Onesie": CAN_WITHDRAW_AND_TRADE,
  "Infernal Pitchfork": CAN_WITHDRAW_AND_TRADE,
  "Corn Onesie": CAN_WITHDRAW_AND_TRADE,
  "Banana Amulet": CAN_WITHDRAW_AND_TRADE,
  "Bee Suit": CAN_WITHDRAW_AND_TRADE,
  "Crimstone Armor": CAN_WITHDRAW_AND_TRADE,
  "Honeycomb Shield": CAN_WITHDRAW_AND_TRADE,
  "Flower Crown": CAN_WITHDRAW_AND_TRADE,
  "Oil Can": CAN_WITHDRAW_AND_TRADE,
  "Tofu Mask": CAN_WITHDRAW_AND_TRADE,
  "Camel Onesie": CAN_WITHDRAW_AND_TRADE,
  "Dev Wrench": CAN_WITHDRAW_AND_TRADE,
  "Infernal Drill": CAN_WITHDRAW_AND_TRADE,
  "Grape Pants": CAN_WITHDRAW_AND_TRADE,
  "Carrot Amulet": CAN_WITHDRAW_AND_TRADE,
```
Some items appear to be missing in frontend

**Review by c-py — DISMISSED** [review]
LGTM! Working well however I was not able to withdraw green amulet as it was missing from the frontend.

**Pull request merged**